showcase-docs/getting-started.md

```diff
@@ -2,6 +2,151 @@
 
 There are several different methods for running the Backstage Showcase app today. We currently have support for running the application locally, using a helm chart to deploy to a cluster, and manifests for deployment using ArgoCD.
 
+## Telemetry collection
+
+The telemetry data collection feature is used to enhance your experience with the application without compromising your privacy. To enable or disable telemetry data collection, you can configure the [`@janus-idp/backstage-plugin-analytics-provider-segment`](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/analytics-provider-segment) plugin accordingly.
+
+- **Anonymous configuration**:
+
+  - IP addresses are anonymized (`maskIP: true`), and recorded as `0.0.0.0`.
+  - `anonymousId` used for tracking is a hash derived from the user's username.
+
+- **Data Collection Overview**:
+  - **Events Tracked**: Page visits, clicks on links or buttons.
+  - **Common Data Points for All Events**:
+    - User-related info: Locale, timezone, userAgent (browser and OS details).
+    - Page-related info: Title, Category, Extension name, URL, path, referrer, search parameters.
+
+The telemetry data will only be used for internal analysis and product improvements. The collected data is analyzed to understand user interactions with the application while maintaining user anonymity and privacy.
+
+To enable or disable telemetry data collection and customize a telemetry destination, see the following sections.
+
+### Enable Telemetry
+
+To turn on the telemetry feature, you must enable the `analytics-provider-segment` plugin either using the Helm Chart or the RHDH Operator.
+
```

Comment:
We need just 2 notes, one in this section and other in the disable telemetry section. 

Note suggestion for the Enable telemetry section
If the `analytics-provider-segment` plugin is already present in your Helm configuration file or ConfigMap file, set the value of the `plugins.disabled` parameter to `false` to quickly start telemetry data collection.

Note suggestion for the disable telemetry section
If the `analytics-provider-segment` plugin is already present in your Helm configuration file or ConfigMap file, set the value of the `plugins.disabled` parameter to `true` to quickly stop telemetry data collection.

Reply:
added